symbolicator/config.example.yml

```diff
@@ -0,0 +1,9 @@
+# See: https://getsentry.github.io/symbolicator/#configuration
+cache_dir: "/data"
+bind: "0.0.0.0:3021"
+logging:
+  level: "warn"
+  enable_backtraces: true
```

Comment:
Setting the default level to `"warn"` makes sense, for performance one could also choose to disable backtraces. If anything, this is the one I'd propose to remove and leave to symbolicator.